### PR TITLE
feat: scaffolded code

### PR DESCRIPTION
**Pull request by Tulun**
-Ran CRA w/ TypeScript template.
-Added in basic folder structure, router, and css lib (tailwind).

### CODE REVIEW

File: src/reportWebVitals.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```
Comment (daywiss) [inline src/reportWebVitals.ts:3]: not familiar with this, what is it?

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx"
```
Comment (daywiss) [inline tsconfig.json:18]: @Gamaranto how does this look?

also do we want to add additional linting / prettifying?

File: crago.config.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// craco.config.js
+module.exports = {
```
Comment (daywiss) [inline crago.config.ts:2]: is this file craco or crago? also not familiar with this

**Review by daywiss — COMMENTED** [review]
looks good to me. @Gamaranto just want you to see if theres anything you would like to pull in based on the claim dapp

File: crago.config.ts (hunk 3 above)
Comment (Tulun) [reply crago.config.ts]: Seems like it was a bridge with a breaking React 17 change. removed as there is a workaround.

File: src/reportWebVitals.ts (hunk 1 above)
Comment (Tulun) [reply src/reportWebVitals.ts:3]: This comes default with CRA. We can remove it if need be. It helps with web reporting and such

File: tsconfig.json (hunk 2 above)
Comment (Tulun) [reply tsconfig.json:18]: We can definitely dive into the documentation what each of these does -- this is just the default configuration from the template.

Comment (Gamaranto) [reply tsconfig.json:18]: The default looks good enough for now. We can change this later if we need more stuff or if we converge to some team wide ts rules.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "voter-dapp",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/core": "^11.0.0",
```
Comment (Gamaranto) [inline package.json:6]: I think `@emotion/core` is not necessary here as it should be included by `@emotion/styled` and `@emotion/react`

**Review by Gamaranto — APPROVED** [review]
Left some minor comments, but overall looks great. LGTM!

File: tailwind.config.js [hunk 5]
```diff
@@ -0,0 +1,856 @@
+const colors = require('tailwindcss/colors')
```
[the remark this answers is not in the document]
Comment (Tulun) [reply tailwind.config.js:1]: I don't mind either way. This was generated with: npx tailwindcss-cli@latest init --full.

We can pair it down to the empty version, or remove whatever.

**Review by daywiss — APPROVED** [review]
looks good

**Pull request merged**